android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/reactmodules/ExposureNotificationsModule.java

```diff
@@ -51,7 +51,7 @@ public void requestExposureNotificationAuthorization(final Promise promise) {
     FutureCallback<Void> callback = new FutureCallback<Void>() {
       @Override
       public void onSuccess(Void result) {
-        promise.resolve((Util.getEnStatusWritableArray(true)));
+        promise.resolve(enabled ? CallbackMessages.EN_STATUS_ACTIVE : CallbackMessages.EN_STATUS_DISABLED);
```

Comment:
This method only calls `onSuccess` if EN was enabled, this line should be:
`promise.resolve(CallbackMessages.EN_STATUS_ACTIVE)`

Reply:
Thanks @juancruzgs updated. Aside from that is it all good?

Reply:
Yes! 💯